c2corg_ui/templates/i18n.html

```diff
@@ -376,5 +376,8 @@
 <span translate>has updated the article</span>
 <span translate>has updated the route</span>
 <span translate>has updated the image</span>
+<span translate>has added photos to the route</span>
+<span translate>has added photos to the outing</span>
+<span translate>has added photos to the waypoint</span>
```

Comment:
Can't you split up the translation string? E.g. only translate "has created a new", "has updated the" and "has added photos to the"?